ctx.go

```diff
@@ -46,6 +46,29 @@ const (
 // userContextKey define the key name for storing context.Context in *fasthttp.RequestCtx
 const userContextKey = "__local_user_context__"
 
+// decoderPoolMap helps to improve BodyParser's, QueryParser's and ReqHeaderParser's performance
+var decoderPoolMap = map[string]*sync.Pool{}
+
+func init() {
+	for _, tag := range []string{queryTag, bodyTag, reqHeaderTag, paramsTag} {
```

Comment:
can you please put this list near the constants for it, so that the next person who adds some kind of parser will also directly recognize that this list needs to be extended

Reply:
OK

Reply:
done